app/services/analytics_events.rb

```diff
@@ -2990,5 +2990,115 @@ def email_sent(action:, ses_message_id:, **extra)
       **extra,
     )
   end
+
+  def idv_doc_auth_welcome_visited(**extra)
```

Comment:
Should we document `analytics_id` for each of these events?

**Edit:** I'm guessing this is what you are referring to in the second pending item in the original PR commment.

Reply:
:10

Reply:
What do you think about merging this before all the events are documented? Do you think that would be a good follow up item?

Reply:
That's fine.